FlagPhoneNumber/FlagPhoneNumberTextField.swift

```diff
@@ -172,7 +172,7 @@ open class FPNTextField: UITextField, UITextFieldDelegate, FPNCountryPickerDeleg
 	}
 
 	@objc private func displayNumberKeyBoard() {
-		keyboardType = .numberPad
+		keyboardType = .phonePad
```

Comment:
I think we can add this line in the setup function and remove it here and in resetKeyBoard function.